AutomatedLab/AutomatedLab.psm1

```diff
@@ -3609,9 +3609,45 @@ function Add-LabVMUserRight
 }
 #endregion function Add-LabVMUserRight
 
+#region New-LabSourcesFolder
+function New-LabSourcesFolder
+{
+    param
+    (
+        [Switch]
+        $Force
+    )
+    $Path = Get-LabSourcesLocationInternal -Local
+
+    if ($Path -and -not $Force)
+    {
+        return (Get-Item -Path $Path)
+    }
+
+    Write-ScreenInfo -Message 'Downloading LabSources from GitHub. This only happens once if no LabSources folder can be found.' -Type Warning
+
+    $temporaryPath = [System.IO.Path]::GetTempFileName().Replace('.tmp','')
+    [void] (New-Item -ItemType Directory -Path $temporaryPath -Force)
+    $archivePath = (Join-Path -Path $temporaryPath -ChildPath 'master.zip')
+
+    Get-LabInternetFile -Uri 'https://github.com/AutomatedLab/AutomatedLab/archive/master.zip' -Path $archivePath -ErrorAction Stop
+    Expand-Archive -Path $archivePath -DestinationPath $temporaryPath
```

Comment:
The sample scripts folder is at a different location and would not be part of LabSources, right? We can either move the folder in the solution or add some code to handle this here.

Reply:
this was only about the bare minimum for everything to work. i will add the rest.

Reply:
Let's move the sample scripts folder into LabSources. That should have been done in the first place.

Reply:
This is not part of this PR and will require an update to the documentation. I would rather do this in a new branch.

Reply:
What I mean is moving the folder in the internal VS solution to match the actual structure that the installer creates. This does not require any update of the documentation.